src/utils/Helper.ts

```
@@ -298,19 +301,22 @@ export const sortClassesMissed = (data: any[], order: string) => {
 
 export const filterAttendancePercentage = (
   data: any[],
-  category: "more" | "between" | "less"
+  category: 'more' | 'between' | 'less'
 ) => {
   return data.filter(({ present_percent }: { present_percent: string }) => {
     const attendance = parseFloat(present_percent);
 
     if (isNaN(attendance)) return false; // Exclude invalid or missing values
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)